backend/src/main/java/de/neuefische/koheis/backend/chractercards/CharacterCardController.java

```diff
@@ -0,0 +1,24 @@
+package de.neuefische.koheis.backend.chractercards;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/character_cards")
+public class CharacterCardController {
+
+    private final CharacterCardService characterCardService;
+
+    public CharacterCardController(CharacterCardService characterCardService){
+        this.characterCardService = characterCardService;
+    }
+
+    @GetMapping("/cards")
```

Comment:
it is more common to use `/api/character_cards` instead of `/api/character_cards/cards` here for retreiving a list

Reply:
Lorenz pointed out the same thing. As I mentioned him, without extra path, it didn't work. I will ask coaches, because I do not understand the reason.